Library/ViewModels/ProjectPamphletViewModel.swift

```diff
@@ -42,17 +42,17 @@ ProjectPamphletViewModelOutputs {
 
   // swiftlint:disable:next function_body_length
   public init() {
-    let configData = Signal.combineLatest(
+    let configDataAndShouldPrefix = Signal.combineLatest(
       self.configDataProperty.signal.skipNil(),
-      Signal.merge(self.viewDidLoadProperty.signal,
-                   self.viewDidAppearAnimated.signal.filter(isTrue).ignoreValues()
+      Signal.merge(self.viewDidLoadProperty.signal.mapConst(true),
+                   self.viewDidAppearAnimated.signal.filter(isTrue).mapConst(false)
```

Comment:
well after a few more minutes of thinking, i guess it's not a problem to change it